lib/Majkl578/NetteAddons/Doctrine2Identity/DI/IdentityExtension.php

```diff
@@ -25,11 +25,11 @@ class IdentityExtension extends CompilerExtension
 {
 	const NAME = 'doctrine2identity';
 
-	public function loadConfiguration()
+	public function beforeCompile()
 	{
 		$builder = $this->getContainerBuilder();
-
-		$builder->getDefinition('nette.userStorage')
+		$builder->removeDefinition($builder->getByType('Nette\Security\IUserStorage') ?: 'nette.userStorage');
```

Comment:
When I didn't remove it before it didn't work and still throwed the exception like before fix.